docs/Fields.md

```diff
@@ -511,11 +511,20 @@ By default, `<ReferenceField>` is sorted by its `source`. To specify another att
 </ReferenceField>
 ```
 
-You can also prevent `<ReferenceField>` from adding link to children by setting `linkType` to `false`.
+You can also prevent `<ReferenceField>` from adding link to children by setting `linkTo` to `false`.
 
 ```jsx
 // No link
-<ReferenceField label="User" source="userId" reference="users" linkType={false}>
+<ReferenceField label="User" source="userId" reference="users" linkTo={false}>
+    <TextField source="name" />
+</ReferenceField>
+```
+
+You can also use a custom `linkTo` function to get a custom path for the children. This function must accept `record` and `reference` as arguments. On previous versions of React-Admin, this prop was named `LinkType`, however you should migrate it to `linkTo`.
```

Comment:
```suggestion
You can also use a custom `linkTo` function to get a custom path for the children. This function must accept `record` and `reference` as arguments.
```

Reply:
This should be in the upgrade guide, not in the doc.

Reply:
Done